aries_cloudagent/connections/base_manager.py

```diff
@@ -4,19 +4,21 @@
 For Connection, DIDExchange and OutOfBand Manager.
 """
 
-from ....core.profile import ProfileSession
-from ....connections.models.conn_record import ConnRecord
-from ....wallet.base import BaseWallet
-from .messages.connection_invitation import ConnectionInvitation
-from ....ledger.base import BaseLedger
-from ....core.error import BaseError
-from ....connections.models.connection_target import ConnectionTarget
-from ....wallet.util import did_key_to_naked
+from ..core.profile import ProfileSession
+from .models.conn_record import ConnRecord
+from ..wallet.base import BaseWallet
+from ..protocols.connections.v1_0.messages.connection_invitation import (
+    ConnectionInvitation,
+)
+from ..ledger.base import BaseLedger
+from ..core.error import BaseError
+from .models.connection_target import ConnectionTarget
+from ..wallet.util import did_key_to_naked
 import logging
 from typing import Sequence, Tuple
```

Comment:
Minor import ordering issue - external package imports should be in a block before internal ones. I'm not sure why flake8 isn't complaining about that.

Reply:
I have changed the order as requested